addon/components/link-to-component.js

```diff
@@ -3,6 +3,7 @@ import { getOwner } from '@ember/application';
 import { set, get } from '@ember/object';
 import { typeOf } from '@ember/utils';
 import { assert } from '@ember/debug';
+import { FEATURES } from '@ember/canary-features';
```

Comment:
This will eventually break, when the feature flag is removed, right?

It seems to be the cleaner solution though and in case of eventual breakage it's easy to fix.